parsec/core/fs/storage/workspace_storage.py

```diff
@@ -294,6 +300,29 @@ async def get_need_sync_entries(self) -> Tuple[Set[EntryID], Set[EntryID]]:
 
     # Manifest interface
 
+    async def _load_workspace_manifest(self) -> None:
+        try:
+            await self.manifest_storage.get_manifest(self.workspace_id)
+
+        except FSLocalMissError:
+            # It is possible to lack the workspace manifest in local (e.g.
+            # the workspace has been created by a third-party and our
+            # device hasn't tried to access it yet).
+            # In such case it is easy to fall back on an empty manifest
+            # which is a good enough aproximation of the very first version
+            # of the manifest (field `created` is invalid, but it will be
+            # correction by the merge during sync).
+            # This trick is not strictly required (the caller must handle missing
+            # file/folder manifests anyway), but it guarantees the workspace
+            # root folder is always consistent (ls/touch/mkdir always works on it).
+            # This is especially important when the workspace is accessed from
+            # file system mountpoint given having a weird error popup when clicking
+            # on the mountpoint from the file explorer really feel like a bug :/
+            manifest = LocalWorkspaceManifest.new_placeholder(
+                author=self.device.device_id, id=self.workspace_id
+            )
```

Comment:
Sounds like a decent fix. The user should then be able to add files and folders into the workspace and successfully merge all of it when it's back online.